Fakebook.Profile/Fakebook.Profile.DataAccess/Fakebook.Profile.DataAccess.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
let's target `net5.0` for all of our projects

Reply:
The project target version was changed to ```net5.0```.

Reply:
when you push changes addressing a comment like that, you can click the 'resolve conversation' button underneath the comment